### PR TITLE
Secure/update sf.net SVN stable URLs

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Migrated from #10843 and #10841, except `exult`, where 
it should be found out what commit ID at <https://github.com/exult/exult.git> belongs to the SVN revision: `url "svn://svn.code.sf.net/p/exult/code/exult/trunk", :revision => 7520`

### CODE REVIEW

File: Formula/vgmstream.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Vgmstream < Formula
   desc "Library for playing streamed audio formats from video games"
   homepage "https://hcs64.com/vgmstream.html"
-  url "http://svn.code.sf.net/p/vgmstream/code", :revision => 1040
+  url "https://gitlab.kode54.net/kode54/vgmstream.git", :tag => "r1040"
```
Comment (ilovezfs) [inline Formula/vgmstream.rb]: :tag requires :revision, but you can actually use a tarball here.
https://gitlab.kode54.net/kode54/vgmstream/repository/archive.tar.gz?ref=r1040
or
https://gitlab.kode54.net/kode54/vgmstream/repository/archive.tar.bz2?ref=r1040

Comment (vszakats) [inline Formula/vgmstream.rb]: Sorry, it was unverified. I've moved this update to #10898 (including the missing `:revision`). Will follow up there.

Comment (ilovezfs) [inline Formula/vgmstream.rb]: no problem

**Comment by vszakats** [comment]
This passes on Sierra and fails on older OS versions due to the known SVN certificate issue.

**Comment by ilovezfs** [comment]
```
  if MacOS.version >= :sierra
    url "good one"
  else
    url "bad one"
  end
```

**Comment by vszakats** [comment]
Agreeing on this, will do it shortly.

**Comment by vszakats** [comment]
I'll finish this later, I need to meet with some IRL :beer: bottles now.

**Comment by vszakats** [comment]
Finished nevertheless.

**Comment by fxcoudert** [comment]
Do we really want to litter formulas with this "if sierra" construct? Annoying… we should handle that in brew itself

**Comment by ilovezfs** [comment]
@fxcoudert see the prior discussion in the other PRs.

**Comment by ilovezfs** [comment]
:green_apple: Thanks!

**Pull request merged**